src/api/app/lib/backend/api/sources/package.rb

```diff
@@ -51,6 +51,18 @@ def self.service(project_name, package_name)
           nil
         end
 
+        # Returns the content of the _multibuild file (if present)
+        # @return [String]
+        def self.multibuild(project_name, package_name)
+          Rails.cache.fetch("multibuild_#{project_name}_#{package_name}") do
```

Comment:
A cache is nice but a cache also need cache invalidation.

In this case the file is controlled by the backend so the backend would need to invalidate the cache. I wouldn't have an idea how...

Reply:
`"multibuild_#{cache_key_with_version}"` would be enough no? The content of this file won't touch without the package container (package model) being changed, right?

Reply:
I don't think the `_multibuild` files changes so often to justify a query to backend everytime we call 
` Package.get_by_project_and_name(target_project_name, target_package_name, follow_multibuild: true)`

Reply:
> The content of this file won't change without the package container (package model) being changed, right?

Not right. The backend will change this file at will.